packages/trader/src/Modules/Trading/Containers/__tests__/chart-widgets.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { ChartBottomWidgets, ChartTopWidgets, DigitsWidget } from '../chart-widgets';
+
+jest.mock('../chart-widgets.tsx', () => ({
+    ...jest.requireActual('../chart-widgets.tsx'),
+    DigitsWidget: () => <div>MockedDigitsWidget</div>,
```

Comment:
@henry-deriv I agree with @kate-deriv ,  please don't mock all the components that we need to test. :) 
Currently, what you do here doesn't make sense because you're replacing the entire component with fake text and then you just expect this fake text to be there. These tests cannot fail.